.github/workflows/build-and-publish-draft.yaml

```diff
@@ -63,10 +64,70 @@ jobs:
           retention-days: ${{ env.ARTIFACT_RETENTION_DAYS }}
 
       - name: Move file
+        if: github.event_name != 'pull_request'
         run: |
           mkdir drafts
           mv drafts.pdf drafts/drafts.pdf
 
+      - name: Install magick
+        if: github.event_name != 'pull_request'
+        run: |
+          sudo add-apt-repository universe
+          sudo apt-get install -qq -y --no-install-recommends libfuse2t64  # FUSE is required for GH runners to run AppImages
+          wget --quiet ${{ env.IMAGEMAGICK_URL }}
+          find . -name "ImageMagick*" -exec mv '{}' magick \;
+          chmod a+x magick
+
+      - name: Adjust map files for monochrome buld
+        if: github.event_name != 'pull_request'
```

Comment:
why not for PRs?

Reply:
I guess you don't need it in PRs, just when merging to the main branch. Just like economy printable in the rewrite.